canonicalwebteam/blog/django/views.py

```diff
@@ -86,6 +90,65 @@ def group(request, slug, template_path):
     return render(request, template_path, context)
 
 
+def archives(request, template_path="blog/archives.html"):
+    try:
+        page = request.GET.get("page", default="1")
+        category = request.GET.get("category", default="")
+        group = request.GET.get("group", default="")
+        month = request.GET.get("month", default="")
+        year = request.GET.get("year", default="")
+
+        groups = categories = []
```

Comment:
You need to split this into two lines otherwise `groups` and `categories` are pointing at the same list.